packages/sdk/tsconfig/tsconfig-main.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
These tsconfig files are an effort to make the 'projects' within the package (i.e. `src` and `test/src`) more naturally linked. Namely that in `test` source code one must not do an `import ../../../../src/x.ts`.

I am not sure that this is actually a good idea. And it is easily reversible.